test-support/helpers/ember-frost-core.js

```diff
@@ -3,7 +3,15 @@ import Ember from 'ember'
 const {$, typeOf} = Ember
 import {$hook} from 'ember-hook'
 
-const assign = Object.assign || Ember.assign || Ember.merge
+const assign = Object.assign || Ember.assign || Ember.merge || function (assignee, assigner) {
```

Comment:
I don't think you want/need to have a polyfill for assign here. It was failing because of lack of support for ember 2.3 in ember-test-utils, which I have since fixed.